Lib/fontTools/feaLib/builder.py

```diff
@@ -1310,30 +1319,38 @@ def build(self):
             self.setLookAheadCoverage_(suffix, st)
             self.setInputCoverage_(input, st)
 
-            st.SubstCount = len([l for l in lookups if l is not None])
+            st.SubstCount = 0
             st.SubstLookupRecord = []
-            for sequenceIndex, l in enumerate(lookups):
-                if l is not None:
-                    if l.lookup_index is None:
-                        raise FeatureLibError('Missing index of the specified '
-                            'lookup, might be a positioning lookup',
-                            self.location)
-                    rec = otTables.SubstLookupRecord()
-                    rec.SequenceIndex = sequenceIndex
-                    rec.LookupListIndex = l.lookup_index
-                    st.SubstLookupRecord.append(rec)
+            for sequenceIndex, lookupList in enumerate(lookups):
+                if lookupList is not None:
+                    if not isinstance(lookupList, list):
+                        # Can happen with synthesised lookups
+                        lookupList = [ lookupList ]
```

Comment:
i'm reluctant to run `black` on existing modules as it creates noise in the git blame/log, but we can at least try to be close to the black (or PEP8) style for the new lines added to existing modules. Sure, whitespace is innocuous, but I still find it distracting.